subprojects/docs/src/samples/user-input/pass-credentials-to-external-tool-via-stdin/tests/gradlePropertiesCredentials.sample.conf

```diff
@@ -0,0 +1,3 @@
+executable: gradle
```

Comment:
I suspect we don't have this feature, but can you specify environment variables to be set in an exemplar check?